src/TurbulenceClosures/turbulence_closure_implementations/isopycnal_skew_symmetric_diffusivity.jl

```diff
@@ -116,36 +116,66 @@ struct FluxTapering{FT}
 end
 
 """
-    taper_factor_ccc(i, j, k, grid::AbstractGrid{FT}, buoyancy, tracers, tapering::FluxTapering) 
+    taper_factor(i, j, k, grid, closure, tracers, buoyancy) 
 
 Return the tapering factor `min(1, Sₘₐₓ² / slope²)`, where `slope² = slope_x² + slope_y²`
-that multiplies all components of the isopycnal slope tensor. All slopes involved in the
-tapering factor are computed at the cell centers.
+that multiplies all components of the isopycnal slope tensor. The tapering factor is calculated on all the
+faces involved in the isopycnal slope tensor calculation. The minimum value of tapering is selected.
```

Comment:
Should we select the minimum, or maximum? Wouldn't maximum be more conservative?

Reply:
@simone-silvestri